Header Header
float32 Force       # The amount of force needed to be exerted on the contact point
float32 Moment      # The amount of torque needed to be exerted on the contact point
geometry_msgs/Pose ContactPoint # The coordinates of the point for contact
